### ISSUE TITLE
NaN Value Error when all data is 0 in Radar Chart

### ISSUE THREAD

**Issue by Marcusneo121** [imaNNeo/fl_chart#1200]
NaN Value Error appear when all the data in Radar Chart is equal to 0. Found similar post on this error, but was indicated that fixed in latest version of FL Chart. Does anyone know any solution?

Error Message 
`The following assertion was thrown during paint():
Offset argument contained a NaN value.
'dart:ui/painting.dart':
Failed assertion: line 43 pos 10: '<optimized out>'
`

**Versions**
 - Flutter 3.3
 - FL Chart version 0.55.2

**Comment by imaNNeoFighT** [comment]
Please provide a reproducible code.

**Comment by ColinSchmale** [comment]
Some additional information that might help:

```
When the exception was thrown, this was the stack
#2      _offsetIsValid (dart:ui/painting.dart:43:10)
#3      Canvas.drawCircle (dart:ui/painting.dart:4695:12)
#4      CanvasWrapper.drawCircle
canvas_wrapper.dart:67
#5      RadarChartPainter.drawDataSets.<anonymous closure>
radar_chart_painter.dart:280
#6      ListMapView.forEach (dart:_internal/list.dart:242:8)
#7      RadarChartPainter.drawDataSets
radar_chart_painter.dart:256
#8      RadarChartPainter.paint
radar_chart_painter.dart:60
#9      RenderRadarChart.paint
radar_chart_renderer.dart:86
```

**Comment by imaNNeo** [comment]
We have fixed this issue: #1217 in 0.60.0
I think this issue is the same. So we can close this issue.
